### PR TITLE
Update free-programming-books-pl.md

### PR DESCRIPTION
**Pull request by danielkrupinski**
Add Java courses and remove not working link.

### CODE REVIEW

File: free-programming-books-pl.md [hunk 1]
```diff
@@ -106,7 +108,6 @@
 ### LaTeX
 
 * [LaTeX kurs](http://www.latex-kurs.x25.pl)
-* [LaTeX. Książka kucharska](http://www.ptm.org.pl/latex-ksiazka-kucharska), Marcin Borkowski, Bartłomiej Przybylski
```
Comment (eshellman) [inline free-programming-books-pl.md]: https://ptm.org.pl/sites/default/files/latex-ksiazka-kucharska.pdf

**Comment by eshellman** [comment]
Looking good! We added a course page for italian today. Just sayin'

**Comment by danielkrupinski** [comment]
Updated.

**Comment by eshellman** [comment]
Super.

**Pull request merged**